spec/controllers/tyre_fees_controller_spec.rb

```diff
@@ -13,22 +13,22 @@
     let!(:tyre_fee2) { FactoryGirl.create(:tyre_fee, garage: garage2) }
     it 'should show only the tyre fees of a given garage' do
       get :index, { garage_id: garage }
-      assigns(:tyre_fees).count.should eq(1)
-      assigns(:tyre_fees).first.should eq(tyre_fee)
+      expect(assigns(:tyre_fees).count).to eq(1)
+      expect(assigns(:tyre_fees).first).to eq(tyre_fee)
     end
   end
 
   describe 'GET new' do
     it 'assigns a new tyre_fee as @tyre_fee' do
       get :new, { garage_id: garage }
-      assigns(:tyre_fee).should be_a_new(TyreFee)
+      expect(assigns(:tyre_fee)).to be_a_new(TyreFee)
     end
   end
 
   describe 'GET edit' do
     it 'assigns the requested tyre_fee as @tyre_fee' do
       get :edit, { garage_id: garage, id: tyre_fee.to_param }
-      assigns(:tyre_fee).should eq(tyre_fee)
+      expect(assigns(:tyre_fee)).to eq(tyre_fee)
```

Comment:
Using `assigns` in test controllers I think is deprecated: https://github.com/rails/rails/issues/18950